spec/features/editing_images/remove_lead_image_spec.rb

```diff
@@ -26,20 +28,20 @@ def given_there_is_an_edition_with_a_lead_image
     @edition = create(:edition,
                       document_type_id: document_type.id,
                       lead_image_revision: @image_revision)
+
+    @publishing_api_request = stub_publishing_api_put_content(@edition.content_id, {})
```

Comment:
Similar point here - by the way I think a convention we've got is setting up the stub near the point of the request which we may be starting to break here. Without doing a bit of an audit I'm not sure how prevalent it is.